torch/onnx/utils.py

```diff
@@ -529,17 +526,12 @@ def _model_to_graph(model, args, verbose=False,
 
 
 def export_to_pretty_string(model, args, f, export_params=True, verbose=False, training=None,
-                            input_names=None, output_names=None, aten=False,
-                            operator_export_type=None, export_type=ExportTypes.PROTOBUF_FILE,
-                            example_outputs=None, google_printer=False,
-                            opset_version=None, _retain_param_name=True,
+                            input_names=None, output_names=None, operator_export_type=None,
+                            export_type=ExportTypes.PROTOBUF_FILE, example_outputs=None,
+                            google_printer=False, opset_version=None, _retain_param_name=True,
                             keep_initializers_as_inputs=None, custom_opsets=None, add_node_names=True,
                             do_constant_folding=True, dynamic_axes=None):
-    if aten:
-        assert operator_export_type is None
-        assert aten
-        operator_export_type = OperatorExportTypes.ONNX_ATEN
-    elif operator_export_type is None:
+    if operator_export_type is None:
```

Comment:
I think it'd be better to just change the default value of the arg to OperatorExportTypes.ONNX (both here and in `export()`).

Reply:
Make sense. But in export(), we may also check if `PYTORCH_ONNX_CAFFE2_BUNDLE` parameter is needed. https://github.com/pytorch/pytorch/blob/27ae5c5ff0ab2ee1d0fc2b401eb614a75b643abb/torch/onnx/utils.py#L80